doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -161,10 +173,10 @@ Additional scenario XML elements:
 ``irq`` under parent of ``vuart``:
   vCOM irq.
 
-``target_vm_id``:
+``target_vm_id`` under parent of ``vuart`` id 1:
```

Comment:
So is it only valid for the `vuart` parent of VM ID #1 (which would be the service VM then?)

Reply:
no, here the parent is vuart id 1, i.e. vuart1.

Reply:
Can we use this phrasing instead then?
```suggestion
``target_vm_id`` under parent of ``vuart1``:
```